packages/components/app/styles/components/reveal.scss

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+//
+// REVEAL COMPONENT
+//
+
+.hds-reveal__toggle-button {
+  min-height: 1.75rem; // 28px, Override default so padding difference will be visible
```

Comment:
I would add a comment here, highlighting that we're overriding some styles of the tertiary button, per design specs